pages/common/dep.md

```diff
@@ -0,0 +1,23 @@
+# dep
+
+> Tool for dependency management in the Go ecosystem.
+
+- Initialize a directory to be a project:
```

Comment:
I'd suggest something like "Initialize a directory as the root of a Go project" instead. Thoughts?

Reply:
I agree with this!